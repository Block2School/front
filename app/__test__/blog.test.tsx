import renderer from 'react-test-renderer'
import React from 'react'
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import { shallow } from 'enzyme';
import { render, screen } from '@testing-library/react';
import BlogCard from '../components/blog/blogCard';


describe('<Blog/>', () => {
  test('should render correctly', () => {
    shallow(<BlogCard id={0} title={''} author={''} markdownUrl={''} publicationDate={0} shortDescription={''} editDate={0}/>)
  });
})