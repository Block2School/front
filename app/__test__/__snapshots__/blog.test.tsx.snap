// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Blog/> match snapshot 1`] = `ShallowWrapper {}`;
